example/App.tsx

```diff
@@ -1,18 +1,38 @@
-import * as React from 'react';
-import { StyleSheet, View, Text, Image, Animated, Button } from 'react-native';
+import React, { useRef, useState } from 'react';
+import { View, Text, Image, Animated, Button } from 'react-native';
 // @ts-ignore
 import { ReactNativeZoomableView } from '@openspacelabs/react-native-zoomable-view';
+import { debounce } from 'lodash';
```

Comment:
It might be a better idea to define a small debounce function than to add this dependency:

```js
function debounce(func, wait, immediate) {
  var timeout;
  return function() {
  	var context = this, args = arguments;
  	clearTimeout(timeout);
  	timeout = setTimeout(function() {
  		timeout = null;
  		if (!immediate) func.apply(context, args);
  	}, wait);
  	if (immediate && !timeout) func.apply(context, args);
  };
}
```

example taken from https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_debounce